exercises/simple-linked-list/linked_list.exs

```diff
@@ -0,0 +1,83 @@
+defmodule LinkedList do
+  @opaque t :: tuple()
+
+  @doc """
+  Construct a new LinkedList
+  """
+  @spec new() :: t
+  def new() do
+    # Your implementation here...
```

Comment:
I don't like these at all. I would prefer not to supply the file at all so the user has to write their own module.

Shall I just remove the comments?